packages/integration_test/CHANGELOG.md

```diff
@@ -0,0 +1,221 @@
+## 0.9.2+2
```

Comment:
What should we do with the version changelog now that this no longer has a version number?

Reply:
I'd like to keep this here for historical purposes, but I don't think we'll keep updating the changelog. I can add a notice.